dev-tools/ls-apis/api-manifest.toml

```diff
@@ -412,7 +400,8 @@ ancestor = "nexus-types"
 client = "dns-service-client"
 evaluation = "bogus"
 note = """
-nexus-types depends on dns-service-client for defining some types.
+Past versions of nexus-types that are still referenced in the dependency tree
+depended on dns-service-client for defining some types.
```

Comment:
So if I remove this rule as well, I get this diff:

```diff
 DNS Server (client: dns-service-client)
+    consumed by: crucible-downstairs (crucible/downstairs)
+    consumed by: crucible-pantry (crucible/pantry)
+    consumed by: ddmd (maghemite/ddmd)
+    consumed by: dpd (dendrite/dpd)
+    consumed by: mgd (maghemite/mgd)
     consumed by: omicron-nexus (omicron/nexus)
     consumed by: omicron-sled-agent (omicron/sled-agent)
+    consumed by: propolis-server (propolis/bin/propolis-server)
```

That's what I based this comment on.